### DISCUSSION TITLE
Cannot download Groot2 for Linux

### DISCUSSION THREAD

**Discussion by roboticswithjulia** [BehaviorTree/BehaviorTree.CPP#754] [Q&A]
Hello! 
When I am trying to donwload the [Groot2](https://www.behaviortree.dev/groot/) interface, it's giving me the following error:
`<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>531Q67CMWDQN422Z</RequestId>
<HostId>5tEnDnXw+wQWnfWXe8yWTKD0tl4vhzQV4xtS2t2156mcThLlalZXe3mslrAycVU4WO1yhikHW/8=</HostId>
</Error>`

Do I have to do something else to be able to download the Basic release of the Groot2?

Thank you for your help,

Júlia

**Comment by facontidavide** [comment]
Hi Julia.

Fixed! Thanks for reporting

**Reply by roboticswithjulia** [reply]
Thank you for the quick fix!
